net-libs/mbedtls/mbedtls-2.7.0.recipe

```diff
@@ -75,9 +83,9 @@ REQUIRES="
 
 PROVIDES_devel="
 	mbedtls${secondaryArchSuffix}_devel = $portVersion
-	devel:libmbedcrypto$secondaryArchSuffix = 0 compat >= 0
-	devel:libmbedtls$secondaryArchSuffix = 10 compat >= 10
-	devel:libmbedx509$secondaryArchSuffix = 0 compat >= 0
+	devel:libmbedcrypto$secondaryArchSuffix = $libcryptoVersionCompat
+	devel:libmbedtls$secondaryArchSuffix = $libtlsVersionCompat
+	devel:libmbedx509$secondaryArchSuffix = $libx509VersionCompat
```

Comment:
Do we use *Compat* here too? I thought it was only needed for "lib:lib*"

Reply:
I also thought, **_some time ago_**, that we should only use `compat` for `lib:lib*` lines.
But later I understood that the `compat` can also be used for `devel:lib*` lines, although this is not automatic.
If I understand correctly, we should only avoid the `compat` when we want to make sure that any external recipe depending on a `devel:libmbed*` would use an explicit version of mbedtls's `devel:libmbed*`.

@alaviss, do you know whether or not we should use  compats in this recipe, and where? Thanks!

FWIW, mbedtls does **_not_** use the `.so.x.y.z` notation for its libs. Also, there are no symlinks.

Reply:
From [BuildingPackages][0]

> The `version_ref` after the "compat"/"compatible" string specifies the oldest version the resolvable is backwards compatible with.

So no, we don't use `compat`, only `lib<name>Version`

[0]: https://github.com/alaviss/haiku/blob/wiki-fixes/docs/develop/packages/BuildingPackages.rst#the-packageinfo